specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/providers.json

```diff
@@ -152,6 +158,29 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "ProviderError": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ProviderErrorBody"
+        }
+      },
+      "x-ms-external": true
+    },
+    "ProviderErrorBody": {
```

Comment:
This is the same as `CloudError`. Does a new model need to be created? Or is the thought they could diverge at some point?

Reply:
They can be the same. So in the preview version, I put the default error model into the common file.

Reply:
They are the same. I updated it to CloudError, as well as the one in secrets.json, which I defined in the last PR

Reply:
@heaths They are the same. I updated it to CloudError, as well as the one in secrets.json, which I defined in the last PR